test/y2partitioner/actions/controllers/filesystem_test.rb

```diff
@@ -307,10 +307,28 @@
     end
 
     RSpec.shared_examples "default_mount_by" do
-      it "sets mount by to the default value" do
-        subject.apply_role
-        expect(subject.filesystem.mount_point.mount_by)
-          .to eq(Y2Storage::StorageManager.instance.configuration.default_mount_by)
+      before do
+        Y2Storage::StorageManager.instance.configuration.default_mount_by = default_mount_by
+      end
+      let(:default_mount_by) { Y2Storage::Filesystems::MountByType::UUID }
```

Comment:
np: you can remove this line as it is defined in each context.

Reply:
Not really. With shared_examples this is required.  That's why I added it.